db/migrate/20230509133628_add_columns_to_employee.rb

```diff
@@ -0,0 +1,15 @@
+class AddColumnsToEmployee < ActiveRecord::Migration[7.0]
+  def change
+    change_table :employees do |t|
+      t.integer :number
```

Comment:
For the future: You can create a table with columns. No need for two separate migrations.